ovirt_imageio/client/_tool.py

```diff
@@ -21,8 +22,11 @@ def main():
     _download.register(parser)
     args = parser.parse()
 
+    is_file = os.path.isfile(args.log_file)
+    if not is_file:
+        logging.StreamHandler(args.log_file)
     logging.basicConfig(
-        filename=args.log_file,
+        filename=args.log_file if is_file else None,
```

Comment:
Since logging uses stream=sys.stderr by default if filename is not set, we can change the default log_file to None instead.